tests/helm/expectedTemplates/worker/metastore-pvc.yaml

```diff
@@ -23,7 +23,7 @@ metadata:
 spec:
   resources:
     requests:
-      storage: 1Gi
+      storage: 256Mi
```

Comment:
if this is over the originally assigned space, how to expand this on restart?

Reply:
Need to set a larger number in the helm config file, so that when restart we will create a new PVC with a larger storage request. K8s will then find an appropriate PV of enough size to match the new PVC.
This only applies when we are using PV/PVC for storage. If directly using hostPath, we basically have the whole disk so if that goes over we are kind of in trouble.